index.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>HTML and CSS Quiz</title>
+    <link rel="stylesheet" href="quiz.css" media="screen" title="no title" charset="utf-8">
+  </head>
+  <body>
+    <div class="product">
+        <article>
+          <header>
+            <h1>Yellow Dunk Card</h1>
+          </header>
+          <div class="section">
+            <img src="https://unsplash.it/250/250/?random=1" alt="The Product" />
```

Comment:
`<img>` tags are sort of the platypus of tags. They take on properties of both inline and block elements. To make sure you have a block level element try wrapping your image in a `<div>` tag.